src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/AssemblyPart.cs

```diff
@@ -41,6 +51,25 @@ public AssemblyPart(Assembly assembly)
         /// <inheritdoc />
         public IEnumerable<TypeInfo> Types => Assembly.DefinedTypes;
 
+        public PrecompiledViews PrecompiledViews
```

Comment:
I have some ideas about how to improve the factoring of this in the future, but let's leave that for another PR.